pkg/wireguard/agent/agent.go

```diff
@@ -480,9 +481,10 @@ func (a *Agent) updatePeerByConfig(p *peerConfig) error {
 	}
 
 	log.WithFields(logrus.Fields{
-		logfields.Endpoint: p.endpoint,
-		logfields.PubKey:   p.pubKey,
-		logfields.IPAddrs:  p.allowedIPs,
+		logfields.Endpoint:                    p.endpoint,
+		logfields.PubKey:                      p.pubKey,
+		logfields.IPAddrs:                     p.allowedIPs,
+		logfields.WireguarPersistentKeepalive: option.Config.WireguarPersistentKeepalive,
```

Comment:
Since the option is static and the same for all peers, I don't think it makes sense to log it here (all other fields here are dynamic). It will just clutter the log lines without providing additional information